src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
```

Comment:
Rename p => person

Reply:
`Math.ceil(p.died / 100) === century`

I would prefer it as separate function for example ```isDateInGivenCentury(someDate, century) {
return Math.ceil(p.died / 100) === century
}```

more readable, easier to test.

FYI, this logic does not work for negative numbers. If it's expected, you should throw an error if such number was provided. 

( Math.ceil(-80 / 100) returns -0, should be -1)